clipper_admin/clipper_manager.py

```diff
@@ -498,13 +501,116 @@ def deploy_model(self,
                 for r in range(num_containers)
             ])
 
+
+
+
+    def deploy_R_model(self,
+                     name,
+                     version,
+                     model_data,
+                     container_name,
+                     labels,
+                     input_type,
+                     num_containers=1):
+        """Registers a model with Clipper and deploys instances of it in containers.
+
+        Parameters
+        ----------
+        name : str
+            The name to assign this model.
+        version : int
+            The version to assign this model.
+        model_data : str or BaseEstimator
+            The trained model to add to Clipper. This can either be a
+            Scikit-Learn trained model object (an instance of BaseEstimator),
+            or a path to a serialized model. Note that many model serialization
+            formats split the model across multiple files (e.g. definition file
+            and weights file or files). If this is the case, `model_data` must be a path
+            to the root of a directory tree containing ALL the needed files.
+            Depending on the model serialization library you use, this may or may not
+            be the path you provided to the serialize method call.
+        container_name : str
+            The Docker container image to use to run this model container.
+        labels : list of str
+            A set of strings annotating the model
+        input_type : str
+            One of "integers", "floats", "doubles", "bytes", or "strings".
+        num_containers : int, optional
+            The number of replicas of the model to create. More replicas can be
+            created later as well. Defaults to 1.
```

Comment:
Fix the method documentation. The description of the method and of the `model_data` argument are both wrong.